templates/zsh.txt

```diff
@@ -98,7 +98,9 @@ if [[ -o zle ]]; then
 
         if [[ "{{ "${#words[@]}" }}" -eq 2 ]]; then
             # Show completions for local directories.
-            _files -/
+            _cd -/
+
+	    # Don't return 0 so we can fall back to another comleter if unsuccessful.
```

Comment:
Avoid this - we don't want to fallback to other completions.

Reply:
We want this here @ajeetdsouza 

Without it local directory completion will work very badly. For instance 

    cd oo<tab>

will not complete `foo`, and instead complete nothing.

Reply:
Won't it also complete things like file names, which we don't want?

Reply:
Zsh fallback is smart :). It first tries completing corrections as directories; if that fails it completes files.

Also, looking at the logic -- the way I had the patch originally `z o<tab>` will fall back to corrections, but `z o <tab>` will not. That's probably what you want, since that's where you do the fzf / history magic. Best to keep `z o<tab>` as close to `cd o<tab>` completion as possible (which is what my patch did).

Reply:
Got it. Fixed this in `main`.